benchmarks/python/core.py

```diff
@@ -222,14 +222,15 @@ def set_metrics(
                     iobytes += out.element_size() * out.numel()
 
         self.benchmark.extra_info["IOBytes"] = iobytes
-        bandwidth_bps = (
-            iobytes * self.benchmark.stats["rounds"]
-        ) / self.benchmark.stats["total"]
-        self.benchmark.extra_info["Bandwidth (Bps)"] = bandwidth_bps
-        self.benchmark.extra_info["Bandwidth (GBps)"] = bandwidth_bps / 1e9
-        self.benchmark.extra_info["% Peak Bandwidth (SOL)"] = (
-            100 * (bandwidth_bps / 1e9) / PEAK_BANDWIDTH_GBPS
-        )
+        if self.has_cuda_event:
```

Comment:
Similar to above, this feels like it will lead to silent errors. Benchmarking inaccurate readings is quite tedious given the number of inputs we run. I would make this an assert instead.